client/termsetting/handlers/numeric.continuous.ts

```diff
@@ -55,6 +55,19 @@ export function getHandler(self) {
 
 			vr.render()
 
+			const convert2ZScoreDiv = div.append('div').style('display', 'inline-block').style('padding', '3px 10px')
+
+			convert2ZScoreDiv
```

Comment:
use `dom/checkbox.js` to create this

Reply:
i agree to have the zscore option in termsetting edit menu. when the zscore box is checked, it shouldn't show the `Scale values` option as it cannot be coapplied to zscore